TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/workflow/commands/configure.py

```diff
@@ -118,16 +164,25 @@ def _save_config(store, new_config: WorkflowConfig, session_name: str):
         raise click.ClickException(f"Failed to save configuration: {e}")
 
 
-def _handle_stdin_edit(store, session_name: str):
+def _handle_stdin_edit(wf_config_store, secret_store, session_name: str):
     """Handle configuration edit from stdin"""
     stdin_stream = click.get_text_stream('stdin')
     stdin_content = stdin_stream.read()
 
     new_config = _parse_config_from_stdin(stdin_content)
-    _save_config(store, new_config, session_name)
+    _save_config(wf_config_store, new_config, session_name)
+
+    existing_secrets, missing_secrets = _get_basic_creds_secrets_in_config(secret_store, new_config)
+    if existing_secrets:
+        click.echo(f"Found {len(existing_secrets)} existing secrets that will be used for HTTP-Basic authentication "
+                   f"of requests to clusters:\n  " + "\n  ".join(existing_secrets))
+    if missing_secrets:
+        raise click.ClickException(f"Found {len(missing_secrets)} missing secrets that still need to be created "
```

Comment:
Similarly, I'd somewhat prefer that clickexceptions are raised only in the `cli.py` or in click command functions.

From this code, we should raise a error that has this message (and ideally it's a pretty specific/custom error type) and then catch that in the command itself to raise this error.